package.json

```diff
@@ -9,9 +9,14 @@
     "pretty-quick": "^3.1.3"
   },
   "scripts": {
+    "requirements": "yarn install --frozen-lockfile && pip install -r requirements-dev.txt",
```

Comment:
Wanted to add so it would be just easier to add if need be, but had own reservations about it as well - mostly about mixing JS and Py.
If we add in poetry and pipx, we declare them as additional dependencies which is a bit of a hassle then - they're brilliant tools, but aren't available in some environments, and it could break compatibility if someone installs fresh. Dependency install step should be as lightweight as it comes imo.
I'll remove the line and will just re-add full commands in Readme.

Reply:
I think your current or updated solution are good, just wanted to think about how we are managing js/py.  Poetry is a heavier if it's just a few depedencies that's for sure.  We could make a comment about how a virtual env is needed.  Everything else looks good!